db/seeds.rb

```diff
@@ -115,24 +115,20 @@
                              configuration_snapshot: 2, description: 'This is a test description.')
 
 # SearchFacet seeds
```

Comment:
I'd add a comment here explaining why we chose these three facets.  Or if it's not particularly important to have these three particular facets, maybe use populate_from_schema instead?

Reply:
The reason: one is string based, but not arrays.  The other is string & array based.  The last is numeric.  This covers all of our facet types.  I'll add a comment to that effect.

Reply:
8b721b0